lib/modules/datasource/glasskube-packages/index.ts

```diff
@@ -0,0 +1,75 @@
+import { cache } from '../../../util/cache/package/decorator';
+import { joinUrlParts } from '../../../util/url';
+import * as glasskubeVersioning from '../../versioning/glasskube';
+import { Datasource } from '../datasource';
+import type { GetReleasesConfig, ReleaseResult } from '../types';
+import {
+  GlasskubePackageManifestYaml,
+  GlasskubePackageVersions,
+  GlasskubePackageVersionsYaml,
+} from './schema';
+
+export class GlasskubePackagesDatasource extends Datasource {
+  static readonly id = 'glasskube-packages';
+  static readonly defaultRegistryUrl =
+    'https://packages.dl.glasskube.dev/packages';
+  override readonly customRegistrySupport = true;
+  override defaultVersioning = glasskubeVersioning.id;
+
+  override defaultRegistryUrls = [
+    GlasskubePackagesDatasource.defaultRegistryUrl,
+  ];
+
+  constructor() {
+    super(GlasskubePackagesDatasource.id);
+  }
+
+  @cache({
+    namespace: `datasource-${GlasskubePackagesDatasource.id}`,
+    key: ({ packageName }: GetReleasesConfig) => `${packageName}`,
```

Comment:
this should include the registry url, otherwise we get caching issues with different registries.